### PR TITLE
[zh] Fix type parameter in example

### PR DESCRIPTION
**Pull request by jsoref**
### Description

* #48769 for Chinese

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi @jsoref , this file has some lines insynced with the english upstream.
Would you double check the differences by running this command on your `main` branch:

```
./scripts/lsync.sh content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md
```
```diff
--- a/content/en/blog/_posts/2023-12-15-volume-attributes-class/index.md
+++ b/content/en/blog/_posts/2023-12-15-volume-attributes-class/index.md
@@ -3,10 +3,10 @@ layout: blog
 title: "Kubernetes 1.29: VolumeAttributesClass for Volume Modification"
 date: 2023-12-15
 slug: kubernetes-1-29-volume-attributes-class
+author: >
+  Sunny Song (Google)
 ---
 
-**Author**: Sunny Song (Google)
-
 The v1.29 release of Kubernetes introduced an alpha feature to support modifying a volume
 by changing the `volumeAttributesClassName` that was specified for a PersistentVolumeClaim (PVC).
 With the feature enabled, Kubernetes can handle updates of volume attributes other than capacity.
@@ -68,7 +68,7 @@ If you would like to see the feature in action and verify it works fine in your
      name: csi-sc-example
    provisioner: pd.csi.storage.gke.io
    parameters:
-     disk-type: "hyperdisk-balanced"
+     type: "hyperdisk-balanced"
    volumeBindingMode: WaitForFirstConsumer
    ```
 
@@ -174,4 +174,4 @@ Special thanks to all the contributors that provided great reviews, shared valua
 *   Jordan Liggitt (liggitt)
 *   Matthew Cary (mattcary)
 *   Michelle Au (msau42)
-*   Xing Yang (xing-yang)
\ No newline at end of file
+*   Xing Yang (xing-yang)
```

**Comment by windsonsea** [comment]
Thanks, please [squash your commits into a single one](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits).

**Comment by windsonsea** [comment]
Thanks, please [squash your commits into a single one](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits).

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 1]
```diff
@@ -11,11 +13,6 @@ date: 2023-12-15
 slug: kubernetes-1-29-volume-attributes-class
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:13]:
```suggestion
slug: kubernetes-1-29-volume-attributes-class
author: >
  Sunny Song (Google)
```

**Review by windsonsea — COMMENTED** [review]
Nit
/approve

**Comment by reylejano** [comment]
/retitle "[zh] Fix type parameter in example"

**Comment by reylejano** [comment]
/retitle [zh] Fix type parameter in example

**Pull request merged**